src/utils/plotSpecs/generateHeatmapSpec.js

```diff
@@ -1,45 +1,45 @@
-const generateSpec = (config, groupName) => {
-  let legend = [];
-  if (config.legend.position === 'horizontal') {
-    legend = [
-      {
-        fill: 'color',
-        type: 'gradient',
-        orient: 'bottom',
-        direction: 'horizontal',
-        title: ['Intensity'],
-        labelFont: { value: config.fontStyle.font },
-        titleFont: { value: config.fontStyle.font },
-        gradientLength: {
-          signal: 'width',
-        },
+const generateSpec = (config) => {
+  let legend = [
+    {
+      fill: 'color',
+      type: 'gradient',
+      orient: 'bottom',
+      direction: 'horizontal',
+      title: ['Intensity'],
+      labelFont: { value: config.fontStyle.font },
+      titleFont: { value: config.fontStyle.font },
+      gradientLength: {
+        signal: 'width',
       },
-      {
-        fill: 'cellSetColors',
-        title: groupName,
-        type: 'symbol',
-        orient: 'top',
-        offset: 40,
-        symbolType: 'square',
-        symbolSize: { value: 200 },
-        encode: {
-          labels: {
-            update: {
-              text: { scale: 'cellSetIDToName', field: 'label' },
-            },
+    },
+    {
+      fill: 'cellSetColors',
+      title: 'Cluster ID',
```

Comment:
We are using the heatmap in data integration with the titles like this: https://docs.google.com/presentation/d/1-6mtMrZF14GRAYtLjh3F7FmhyE4o5WRpwyxt5gISGDo/edit#slide=id.gae6b21872b_1_12

(with coloring by sample), before it was set up to allow for the title to be set according to what we need, is this causing problems on that side?

Reply:
I didn't get what you mean, I don't see a heatmap in the slides?